README.md

```diff
@@ -103,13 +103,17 @@ Here's an example of deployment as a 3 tier application using the following imag
  ```
  docker run --name alfio-db -e POSTGRES_DB=postgres -e POSTGRES_USERNAME=postgres -e POSTGRES_PASSWORD=alfiopassword --restart=always -d -v /path/to/local/data:/var/lib/postgresql/data postgres
  ```
-    * Note: on Mac volumes don't work (see https://jhipster.github.io/installation.html for a possible workaround), launch the above command without the `-v` parameter (data are lost at every restart)
+    
+Note: On Mac, volumes don't work (see https://jhipster.github.io/installation.html for a possible workaround), launch the above command without the `-v` parameter (data are lost at every restart).
 
  * Launch the alf.io server
  ```
- docker run --name alfio --link alfio-db:postgres -d alfio/alf.io
+ docker run --name alfio --link alfio-db:postgres -p 8080:8080 -p 8443:8443 -e "POSTGRES_ENV_USE_SSL=disable" -d alfio/alf.io
```

Comment:
I don't think that this is something we want to do. It is not consistent with our security policy, and we're very strict about it.
The rule is: if you've a proper production PostgreSQL instance, you have also SSL support. Otherwise it's not a proper PROD instance for us :-)

If you want, you can still do it (your server, your choice). But we shouldn't suggest it on our README file.